bukkit/src/main/java/io/github/kosmx/emotes/bukkit/BukkitWrapper.java

```diff
@@ -85,6 +90,29 @@ private void receivePluginMessage(String ignore, Player player, byte[] message){
                     player.sendPluginMessage(this, Emotepacket, new EmotePacket.Builder().configureToSendStop(data.emoteData.hashCode(), player.getUniqueId()).build().write().array());
                 }
                 else {
+                    if(this.config.getBoolean("slownessWhilePlaying.enabled")) {
+                        if(data.emoteData != null) {
+                            if(!data.emoteData.isInfinite) {
+                                Bukkit.getScheduler().runTaskLater(this, () -> {
```

Comment:
This scheduled task may cause issues, when a player starts an emote, then stops earlier, then starts another.
I'll reimplement it, but it's mostly fine for now.